test/fixtures/cookbooks/test/recipes/package.rb

```diff
@@ -3,14 +3,14 @@
 
 case node['platform_family']
 when 'amazon', 'rhel'
```

Comment:
Do we just need to remove `amazon` here? We aren't providing the IUS repo to `amazon in the install code, so not sure why we would look for the package for `haproxy18u`